ExpressFirstApp/tests/todo.test.js

```diff
@@ -99,6 +99,61 @@ describe('POST /todo', () => {
 });
 
 describe('DELETE /todo', () => {
+  it('should throw an error while trying to delete tasks', async () => {
+    const spyOnReadFile = jest.spyOn(fs, 'readFile').mockImplementation((path, encoding, callback) => {
+      callback(new Error('Could not read the file content.'))
+    });
+
+    const taskIdToDelete = '1';
+
+    const response = await request(app)
+      .delete('/todo')
+      .send({ todoName, id: taskIdToDelete });
+
+    expect(response.status).toBe(HttpStatusCodes.INTERNAL_SERVER_ERROR);
+    expect(response.error.text).toBe('Could not read the file content.');
+    expect(spyOnReadFile).toHaveBeenCalled();
+
+    spyOnReadFile.mockRestore();
+  });
+
+  it('should delete a task by given todoName and id and return OK', async () => {
```

Comment:
in the tests you are most probably testing the expected behaviour, so the main goal is not to receive OK response, but to have the task completed. So please think about better naming :)